packages/contextual-menu/contextual-menu.scss

```diff
@@ -0,0 +1,92 @@
+@use "~@fremtind/jkl-core/jkl";
+@use "contextual-menu-item";
+@use "contextual-menu-trigger-icon";
+
+@include jkl.light-mode-variables {
+    --jkl-contextual-menu--border-color: #{jkl.$color-skifer};
+    --jkl-contextual-menu-bg-color: #{jkl.$color-hvit};
+    --jkl-contextual-menu-item-hover-background-color: #{jkl.$color-sand};
+    --jkl-contextual-menu-item-trigger-font-color: #{jkl.$color-svart};
+}
+
+@include jkl.dark-mode-variables {
+    --jkl-contextual-menu--border-color: #{jkl.$color-hvit};
+    --jkl-contextual-menu-bg-color: #{jkl.$color-svart};
+    --jkl-contextual-menu-item-hover-background-color: #{jkl.$color-skifer};
+    --jkl-contextual-menu-item-border: 2px solid #{jkl.$color-snohvit};
+    --jkl-contextual-menu-item-trigger-font-color: #{jkl.$color-hvit};
+}
+
+@include jkl.comfortable-density-variables {
+    --jkl-contextual-menu-item-font-size: jkl.$typography-font-size-3;
+    --jkl-contextual-menu-item-font-weight: jkl.$typography-weight-normal;
+    --jkl-contextual-menu-menu-item-height: jkl.rem(48px);
+}
+
+@include jkl.compact-density-variables {
+    --jkl-contextual-menu-item-font-size: jkl.$typography-font-size-1;
+    --jkl-contextual-menu-item-font-weight: jkl.$typography-weight-normal;
+    --jkl-contextual-menu-menu-item-height: jkl.rem(32px);
+}
+
+.jkl-contextual-menu {
+    width: fit-content;
+    display: flex;
+    align-items: center;
+    max-width: jkl.rem(300px);
+    margin: 0;
+
+    &__trigger-btn {
+        @include jkl.reset-outline;
+        @include jkl.motion("exit", "snappy");
+        transition-property: color;
+        color: var(--jkl-contextual-menu-item-trigger-font-color);
+        padding: 0;
+        background-color: transparent;
+        user-select: none;
+        max-width: jkl.rem(300px);
+        width: fit-content;
+        height: fit-content;
+
+        &:focus,
+        &:hover {
+            cursor: pointer;
+        }
+    }
+
+    &__menu {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        flex-wrap: nowrap;
+        background-color: var(--jkl-contextual-menu-bg-color);
+        padding-top: jkl.rem(8px);
+        padding-bottom: jkl.rem(8px);
+        box-shadow: 0 jkl.rem(4px) jkl.rem(20px) rgb(49 48 48 / 20%);
+        border-radius: jkl.rem(2px);
+        border: var(--jkl-contextual-menu-item-border);
+        z-index: jkl.$z-index--floating;
+    }
+
+    &__menu-item {
+        display: flex;
+        align-items: center;
+        width: jkl.rem(300px);
+        box-sizing: border-box;
+        user-select: none;
+        font-size: var(--jkl-contextual-menu-item-font-size);
+        font-weight: var(--jkl-contextual-menu-item-font-weight);
+        height: var(--jkl-contextual-menu-menu-item-height);
+
+        &:hover:not(:disabled) {
+            background-color: var(--jkl-contextual-menu-item-hover-background-color);
+            cursor: pointer;
+        }
```

Comment:
Mulig vi trenger en liknende styling som for hover, men ved `:focus` (for tastaturbrukere) – også hvis disabled.